packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -347,6 +347,7 @@ export function enrichOdspError(
 		}
 	}
 	error.addTelemetryProperties(props);
+	error.endpointReachable = response !== undefined;
```

Comment:
This API is called from createOdspNetworkError() only. See my earlier comment that I feel like createOdspNetworkError() should just set this property to true always, as in all callsites we are processing error that we got from service, i.e. it was reachable.